README.md

```diff
@@ -278,6 +284,11 @@ represents a sample library. Each read is counted towards all the annotated
 miRNA species it aligns to, with 1/n, where n is the number of genomic and/or
 transcriptomic loci that read aligns to.
 
+5. **OPTIONAL**. ASCII-style pileups of read alignments produced for individual
+libraries, combinations of libraries and/or all libraries of a given run. The
+exact number and nature of the outputs depends on the workflow
+inputs/parameters.
```

Comment:
Perhaps this could be linked to the section where you describe this in more detail?

Reply:
Will it be enough to link it to the "pileups subworkflow" section in the `pipeline_documentation.md` or should I link it there and add a small explanation prior to the first rule of the workflow?

Reply:
Yes, that's totally fine.